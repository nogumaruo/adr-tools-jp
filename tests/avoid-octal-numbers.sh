adr new First Decision
adr new Second Decision
adr new Third Decision
adr new Fourth Decision
adr new Fifth Decision
adr new Sixth Decision
adr new Seventh Decision
adr new Eighth Decision
adr new Ninth Decision
ls doc/adr
head -8 doc/adr/009-ninth-decision.md
