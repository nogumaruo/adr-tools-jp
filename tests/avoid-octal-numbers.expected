adr new First Decision
doc/adr/001-first-decision.md
adr new Second Decision
doc/adr/002-second-decision.md
adr new Third Decision
doc/adr/003-third-decision.md
adr new Fourth Decision
doc/adr/004-fourth-decision.md
adr new Fifth Decision
doc/adr/005-fifth-decision.md
adr new Sixth Decision
doc/adr/006-sixth-decision.md
adr new Seventh Decision
doc/adr/007-seventh-decision.md
adr new Eighth Decision
doc/adr/008-eighth-decision.md
adr new Ninth Decision
doc/adr/009-ninth-decision.md
ls doc/adr
001-first-decision.md
002-second-decision.md
003-third-decision.md
004-fourth-decision.md
005-fifth-decision.md
006-sixth-decision.md
007-seventh-decision.md
008-eighth-decision.md
009-ninth-decision.md
head -9 doc/adr/009-ninth-decision.md
# 9. Ninth Decision

日付: 1992-01-12

決定者: @xxx, @xxx

## ステータス

提案中
